docker/Dockerfile

```diff
@@ -0,0 +1,102 @@
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of the IoT Agent for the Ultralight 2.0 protocol (IOTAManager) component
+#
+# IOTAManager is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# IOTAManager is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with IOTAManager.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with: [daniel.moranjimenez@telefonica.com]
+
+ARG NODE_VERSION=10.19.0-slim
```

Comment:
Maybe this should be included in the CNR:

```
- Upgrade NodeJS version from 10.17.0 to 10.19.0 in Dockerfile
```

Reply:
fixed in https://github.com/telefonicaid/iotagent-manager/pull/203/commits/3e91cd846fbbdbdc12be9371bc714f98984edd15